pkg/dashboard/resource/resource.go

```diff
@@ -18,6 +18,7 @@ package resource
 
 import (
 	"context"
+	"github.com/nuclio/nuclio/pkg/common/headers"
```

Comment:
Imports order is as follows (with a black line separator between them):
1. built-in modules
2. internal modules ("github.com/nuclio/nuclio/pkg/blablabla")
3. external modules